agent/src/main/java/org/openremote/agent/protocol/http/AbstractHTTPServerProtocol.java

```diff
@@ -81,6 +81,14 @@ public DeploymentInstance(DeploymentInfo deploymentInfo, WebService.RequestHandl
             this.deploymentInfo = deploymentInfo;
             this.requestHandler = requestHandler;
         }
+
+        public DeploymentInfo getDeploymentInfo() {
+            return deploymentInfo;
+        }
+
+        public WebService.RequestHandler getRequestHandler() {
+            return requestHandler;
+        }
```

Comment:
Don't want dependencies on  `AbstractHTTPServerProtocol` for creating servlet deployments in GOPACS, can WebService addServletDeployment/removeServletDeployment methods be utilised?